pkg/services/gitops/install_test.go

```diff
@@ -56,18 +76,46 @@ var _ = Describe("Install", func() {
 		Expect(os.RemoveAll(dir)).To(Succeed())
 	})
 
-	It("checks cluster status", func() {
-		kubeClient.GetClusterStatusStub = func(c context.Context) kube.ClusterStatus {
-			return kube.FluxInstalled
-		}
-		_, err := gitopsSrv.Install(installParams)
-		Expect(err).Should(MatchError("Weave GitOps does not yet support installation onto a cluster that is using Flux.\nPlease uninstall flux before proceeding:\n  $ flux uninstall"))
+	Context("validate install", func() {
```

Comment:
Please add a test for multiple namespaces with the label

Reply:
that case doesn't exist the goal of this PR is to guarantee that.